### ISSUE TITLE
Completely overhaul the UI

### ISSUE THREAD

**Issue by aendrew** [times/axisJS#54]
Everyone loves what axisJS can do. Everyone hates its Bootstrappy UI. 

I'm still of multiple minds as to what's the best way to layout the UI given the sheer number of controls squished into it, but something should be done.

**Comment by nickpetrie** [comment]
Let's talk to Hollie about this please, a consistent approach between all projects will be best

**Comment by aendrew** [comment]
@nickpetrie Yep, that's the plan! :+1: I'm just logging it as an issue to facilitate development once it gets to that stage.

**Comment by aendrew** [comment]
Did quite a lot of work to improve the UI in 2bf3351dada16f6ed62f822467121d52cd0e31b0. It's actually probably pretty passable now. v2 will be nicer, but I'm pretty okay with it at the moment.
